dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.jaxb-3.0.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jakarta.jaxb-3.0
+visibility=private
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+ install
+IBM-Process-Types: client, \
+ server
+Subsystem-Name: Jakarta XML Bindings 3.0
+-features=\
+  com.ibm.websphere.appserver.classloading-1.0, \
+  io.openliberty.jakarta.activation-2.0
+-bundles=\
+  io.openliberty.jakarta.jaxb.3.0; location:="dev/api/spec/,lib/"; mavenCoordinates="jakarta.xml.bind:jakarta.xml.bind-api:3.0.0-RC3", \
+  io.open.jaxb.3.0.internal.tools
```

Comment:
This should be io.openliberty not io.open

Reply:
Shouldn't the io.openliberty.jaxb.3.0.internal.tools bundle be in the public jaxb-3.0 feature instead of this feature?